### PR TITLE
Enables ceph deployment.

### PR DESCRIPTION
**Pull request by dguitarbite**
Since the crowbar support for ceph was incomplete, deploying ceph was autoatically disabled. This PR allows ceph to be deployed.

https://github.com/crowbar/barclamp-provisioner/pull/35://github.com/crowbar/barclamp-provisioner/pull/359

### CODE REVIEW

**Comment by dirkmueller** [comment]
@dguitarbite this is not complete. you need to change allocation of all but the first node to sle12 ..

**Comment by dirkmueller** [comment]
@dguitarbite needs rebase

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ export cinder_conf_volume_type=${cinder_conf_volume_type:-""}
 export cinder_conf_volume_params=${cinder_conf_volume_params:-""}
 export localreposdir_target=${localreposdir_target:-""}
 export want_ipmi=${want_ipmi:-false}
+export wantceph=${wantceph:-}
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: not needed anymore..

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -870,15 +871,30 @@ function onadmin_allocate()
...
-
-        knife node show -F json $computenode > $t
-        json-edit $t -a normal.crowbar_wall.intended_role -v "compute"
-        json-edit $t -a normal.target_platform -v "suse-12.0"
-        knife node from file $t
-        rm -f $t
+
+        if [ -n "$wantceph" ] ; then
+            echo "Setting last two nodes to SLE12 Storage..."
+            local storagenodes=$(crowbar machines list | sort | grep ^d | tail -n 2)
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: LC_ALL=C sort

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -870,15 +871,30 @@ function onadmin_allocate()
...
+
+            for storagenode in $storagenodes; do
+                local t=$(mktemp).json
+                knife node show -F json $storagenode > $t
+                json-edit $t -a normal.crowbar_wall.intended_role -v "storage"
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: you probably want to set the last node to become a compute node nontheless..

**Comment by dirkmueller** [comment]
+1

**Comment by bmwiedemann** [comment]
merged - slightly modified to keep the order of allocate operation

**Pull request closed without merging**